modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,954 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::TranscriptSelection;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
```

Comment:
Not sure how it happened but I forgot to ask for the removal of HiveGeneric_conf as WHEN or INPUT_PLUS is not used